docs/pages/reference/helm-reference/teleport-cluster.mdx

```diff
@@ -1822,6 +1822,14 @@ A `postStart` lifecycle handler to be configured on the main Teleport container.
 Resource requests/limits which should be configured for Teleport containers. These resource limits will also be
 applied to `initContainers`.
 
+<Admonition type="danger">
+Setting CPU limits is an antipattern and is harmful in most cases. Unless you enabled
+[the Static CPU management policy](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy),
+a multithreaded workload with CPU limits will very likely not behave the way you expect when approaching its CPU limit.
+
+Teleport will become unstable once throttling starts. We recommend not to set CPU limits.
```

Comment:
Should we add a paragraph about the implications of such actions?
Since people don't seem to know how it works, it's probably good to give them an idea that CPU limits control the CPU time of the process and not the actual CPU cores reserved. This leads to huge latencies because Teleport will quickly consume its quota and will not be scheduled on any cores for long periods of time.

Reply:
I added a link to this PR.

Reply:
From prev experience, no one will read it.